packages/sourcecred/src/ui/components/AppBar.js

```diff
@@ -137,6 +138,7 @@ const AppBar = (props: Props): React.Node => {
             children
           )}
           <LoadingIndicator />
+          <LoginButton />
```

Comment:
You'll find an `isDev` flag in AdminApp.js

Would you pass it to this module and && it with the button? You can test by adding the "?dev=true" URL param

Reply:
Addressed